### PR TITLE
first portion of updated system settings

### PR DESCRIPTION
**Pull request by ant-tar**
## Description

this PR contains some new system settings and updated old ones, plus some small fixed issues which I noticed along the way 

## Affected versions

2.x, 3.x

## Relevant issues

https://github.com/modxorg/Docs/issues/293

### CODE REVIEW

File: en/building-sites/settings/cache_db_session.md [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
+**Type**: Yes/No  
+**Default**: No  
+**Available In**: Revolution 2.1.1+
 
-**Name**: Enable Database Session Cache
-**Type**: Yes/No
-**Default**: No
-
-When enabled, and [cache\_db](building-sites/settings/cache_db "cache_db") is enabled, database sessions will be cached in the database result-set cache. This means data written to the **$\_SESSION** (e.g. by plugins or snippets) will get cached.
+When enabled, and [cache\_db](building-sites/settings/cache_db "cache_db") is enabled, database sessions will be cached in the database result-set cache. This means data written to the `$\_SESSION` (e.g. by plugins or snippets) will get cached.
```
Comment (Ibochkarev) [inline en/building-sites/settings/cache_db_session.md]:
```suggestion
When enabled, and [cache_db](building-sites/settings/cache_db "cache_db") is enabled, database sessions will be cached in the database result-set cache. This means data written to the `$_SESSION` (e.g. by plugins or snippets) will get cached.
```

File: en/building-sites/settings/custom_resource_classes.md [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
+**Type**: String  
+**Default**:  
+**Removed in** 2.2.0
 
-**Name**: Custom Resource Classes
-**Type**: String
-**Default**:
-
-A comma-separated list of custom Resource classes. Specify with lowercase\_lexicon\_key:className (Ex: wiki\_resource:WikiResource). All custom Resource classes must extend modResource. To specify the controller location for each class, add a setting with \[nameOfClassLowercase\]\_delegate\_path with the directory path of the create/update php files. Ex: wikiresource\_delegate\_path for a class WikiResource that extends modResource.
+A comma-separated list of custom Resource classes. Specify with `lowercase\_lexicon\_key:className` (Ex: `wiki\_resource:WikiResource`). All custom Resource classes must extend modResource. To specify the controller location for each class, add a setting with \[nameOfClassLowercase\]\_delegate\_path with the directory path of the create/update php files. Ex: `wikiresource\_delegate\_path` for a class WikiResource that extends modResource.
```
Comment (Ibochkarev) [inline en/building-sites/settings/custom_resource_classes.md]:
```suggestion
A comma-separated list of custom Resource classes. Specify with `lowercase_lexicon_key:className` (Ex: `wiki_resource:WikiResource`). All custom Resource classes must extend modResource. To specify the controller location for each class, add a setting with `[nameOfClassLowercase]_delegate_path` with the directory path of the create/update php files. Ex: `wikiresource_delegate_path` for a class WikiResource that extends modResource.
```

File: en/building-sites/settings/index.md [hunk 3]
```diff
@@ -216,20 +217,27 @@ A description of each setting follows:
 1. [access_resource_group_enabled](building-sites/settings/access_resource_group_enable)
 1. [allow_duplicate_alias](building-sites/settings/allow_duplicate_alias)
 1. [allow_forward_across_contexts](building-sites/settings/allow_forward_across_contexts)
+1. [allow_manager_login_forgot_password](building-sites/settings/allow_manager_login_forgot_password)
```
Comment (Ibochkarev) [inline en/building-sites/settings/index.md:220]: Lost numbering

File: ru/building-sites/settings/index.md [hunk 4]
```diff
@@ -215,20 +216,27 @@ MODX заполняет значения для имени и описания 
 1. [access_resource_group_enabled](building-sites/settings/access_resource_group_enable)
 1. [allow_duplicate_alias](building-sites/settings/allow_duplicate_alias)
 1. [allow_forward_across_contexts](building-sites/settings/allow_forward_across_contexts)
+1. [allow_manager_login_forgot_password](building-sites/settings/allow_manager_login_forgot_password)
```
Comment (Ibochkarev) [inline ru/building-sites/settings/index.md:219]: Lost numbering

**Review by Ibochkarev — CHANGES_REQUESTED** [review]

File: ru/building-sites/settings/index.md (hunk 4 above)
Comment (ant-tar) [reply ru/building-sites/settings/index.md:219]: in such long lists it is difficult to maintain numbering when inserted manually(with appropriate sortinh) .. however, markdown supports auto numbering(use 1 for each item), what is a big help.

File: en/building-sites/settings/index.md (hunk 3 above)
Comment (ant-tar) [reply en/building-sites/settings/index.md:220]: in such long lists it is difficult to maintain numbering when inserted manually(with appropriate sortinh) .. however, markdown supports auto numbering(use 1 for each item), what is a big help.

**Comment by Ibochkarev** [comment]
@himurovich Thanks!

**Pull request merged**